### DISCUSSION TITLE
compatible with solenso SOL-H400 ??

### DISCUSSION THREAD

**Discussion by dimgod59** [tbnobody/OpenDTU#508] [Q&A]
### Is your feature request related to a problem? Please describe.

Hello,
Do you know if OPEN DTU works with SOLENSO type SOL-H400 micro inverters?
and can it work at the same time with a HOYMILLES 400 micro inverter?
Thanking you in advance.

Best.

Bonjour, 
Savez vous si OPEN DTU fonctionne avec les micro onduleur de la marque SOLENSO type SOL-H400 ? 
et peut il fonctionner en meme temps avec un micro onduleur HOYMILLES 400 ?
Vous remerciant par avance.

Cordialement.

### Describe the solution you'd like

yes or no or alternative OPEN DTU :)

### Additional context

i have already micro inverters HOYMILLES 400

**Comment by tbnobody** [comment]
could be that it's working. could you tell me the first (from left to right) 4 digits of the serial number?

**Comment by dimgod59** [comment]
I do not yet have a panel with a Solenso micro inverter.  the first 4 digits correspond to the month and year of manufacture?  I have a friend who has one on a photo I find type numbers 1121xxxxxx can this be enough?

**Comment by tbnobody** [comment]
1121 sounds like a one channel inverter. And due to the fact that the serial number scheme matches the hoymiles scheme it's most likely that OpenDTU will work.
(I will change this issue to a conversation as its neither a issue nor a feature request)

**Comment by mathieucarbou** [comment]
@tbnobody : in the case of Solenso, what would be the brand and model displayed by OpenDTU when entering the S/N then ?

I only have Hoymiles here at the moment but I am interesting in this discussion because it is possible that I mix some of the inverters with some Solenso.

Thanks!

**Reply by tbnobody** [reply]
I don't know, but just test it? In the inverter overview it will show something like HM-<something> as this text is only determined based on the serial number itself.

**Reply by mathieucarbou** [reply]
How ? I do not have any Solenso inverter nor S/N... You mean that Solenso inverters will be shown as Hoymiles ones ?

**Reply by tbnobody** [reply]
Hm so how should I answer this question? :-) I own only one Hoymiles inverter. I don't even know the brand "Solenso". But as I said, the Text of the inverter type is just derived from the serial number and the serial numbers between Hoymiles and Solenso are most likely the same structure.

**Reply by mathieucarbou** [reply]
Got it ;-) Thanks! 
I might have a lead to be able to find someone to test OpenDTU with Solenso.
I will report back if it happens.

**Reply by mathieucarbou** [reply]
It was quick ;-) 

I got confirmation from someone having a mix of Hoymiles and Solenso inverters that all work fine with OopenDTU. 
Solenso ones are seen as Hoymiles HM-300,350,400 for the type.

See french discussion on FB at: https://www.facebook.com/groups/sunologists/posts/726625425711627/

**Reply by tbnobody** [reply]
Do you have a List of inverters? Currently I know 
* Solenso SOL-H400 

I would add them to the list of compatible devices.
